### ISSUE TITLE
is not working,UINSServiceViewController requires Marzipan

### ISSUE THREAD

**Issue by zhuamaodeyu** [DeVaukz/MachO-Explorer#3]
```
2020-03-19 18:45:43.350509+0800 MachOExplorer[72513:9264819] *** Assertion failure in +[UINSServiceViewController initialize], /BuildRoot/Library/Caches/com.apple.xbs/Sources/ViewBridge/ViewBridge-464.1/UINSServiceViewController.m:203
2020-03-19 18:45:43.350695+0800 MachOExplorer[72513:9264819] [General] UINSServiceViewController requires Marzipan
2020-03-19 18:45:43.353536+0800 MachOExplorer[72513:9264819] [General] (
	0   CoreFoundation                      0x00007fff38ac38ab __exceptionPreprocess + 250
	1   libobjc.A.dylib                     0x00007fff6ebe4805 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff38aecd10 +[NSException raise:format:arguments:] + 88
	3   Foundation                          0x00007fff3b1e5241 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 191
	4   ViewBridge                          0x00007fff6b2f4d25 +[UINSServiceViewController initialize] + 175
	5   libobjc.A.dylib                     0x00007fff6ebe6985 CALLING_SOME_+initialize_METHOD + 17
	6   libobjc.A.dylib                     0x00007fff6ebe72bc initializeNonMetaClass + 638
	7   libobjc.A.dylib                     0x00007fff6ebe7991 _ZL24initializeAndMaybeRelockP10objc_classP11objc_objectR8mutex_ttILb0EEb + 214
	8   libobjc.A.dylib                     0x00007fff6ebd93db lookUpImpOrForward + 969
	9   libobjc.A.dylib                     0x00007fff6ebd8b99 _objc_msgSend_uncached + 73
	10  MachOKit                            0x000000010051b1ab +[MKNode subclasses] + 571
	11  MachOKit                            0x000000010051b425 +[MKNode bestSubclassWithRanking:] + 37
	12  MachOKit                            0x0000000100498ba3 +[MKLoadCommand classForCommandID:] + 99
	13  MachOKit                            0x0000000100499454 +[MKLoadCommand loadCommandAtOffset:fromParent:error:] + 500
	14  MachOKit                            0x00000001004fa62a -[MKMachOImage initWithName:flags:atAddress:inMapping:error:] + 2426
	15  MachOExplorer                       0x000000010001eb27 $sSo12MKMachOImageC4name5flags9atAddress9inMappingABSPys4Int8VGSg_So0aB5FlagsVs6UInt64VSo11MKMemoryMapCtKcfcTO + 183
	16  MachOExplorer                       0x000000010001e221 $sSo12MKMachOImageC4name5flags9atAddress9inMappingABSPys4Int8VGSg_So0aB5FlagsVs6UInt64VSo11MKMemoryMapCtKcfC + 81
	17  MachOExplorer                       0x000000010001d4ab $s13MachOExplorer0A9ODocumentC4read4from6ofTypey10Foundation3URLV_SStKF + 971
	18  MachOExplorer                       0x000000010001e3bc $s13MachOExplorer0A9ODocumentC4read4from6ofTypey10Foundation3URLV_SStKFTo + 236
	19  AppKit                              0x00007fff35f4cdb9 -[NSDocument _initWithContentsOfURL:ofType:error:] + 172
	20  AppKit                              0x00007fff35f4cca2 -[NSDocument initWithContentsOfURL:ofType:error:] + 231
	21  AppKit                              0x00007fff35fbb68a -[NSDocumentController makeDocumentWithContentsOfURL:ofType:error:] + 619
	22  AppKit                              0x00007fff361b691c __97-[NSDocumentController makeDocumentWithContentsOfURL:alternateContents:ofType:completionHandler:]_block_invoke + 91
	23  AppKit                              0x00007fff361b68b6 -[NSDocumentController makeDocumentWithContentsOfURL:alternateContents:ofType:completionHandler:] + 160
	24  AppKit                              0x00007fff35fba8e2 __80-[NSDocumentController openDocumentWithContentsOfURL:display:completionHandler:]_block_invoke + 839
	25  AppKit                              0x00007fff361b5ab8 __144-[NSDocumentController _coordinateReadingAndGetAlternateContentsForOpeningDocumentAtURL:resolvingSymlinks:thenContinueOnMainThreadWithAccessor:]_block_invoke_4 + 31
	26  AppKit                              0x00007fff361b5e0e __144-[NSDocumentController _coordinateReadingAndGetAlternateContentsForOpeningDocumentAtURL:resolvingSymlinks:thenContinueOnMainThreadWithAccessor:]_block_invoke_2.872 + 177
	27  AppKit                              0x00007fff361b5cf7 __144-[NSDocumentController _coordinateReadingAndGetAlternateContentsForOpeningDocumentAtURL:resolvingSymlinks:thenContinueOnMainThreadWithAccessor:]_block_invoke.871 + 153
	28  AppKit                              0x00007fff361b5c0b __144-[NSDocumentController _coordinateReadingAndGetAlternateContentsForOpeningDocumentAtURL:resolvingSymlinks:thenContinueOnMainThreadWithAccessor:]_block_invoke.869 + 243
	29  AppKit                              0x00007fff361c22bf ___NSMainRunLoopPerformBlockInModes_block_invoke + 25
	30  CoreFoundation                      0x00007fff38a477ab __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ + 12
	31  CoreFoundation                      0x00007fff38a476ed __CFRunLoopDoBlocks + 379
	32  CoreFoundation                      0x00007fff38a46d30 __CFRunLoopRun + 2792
	33  CoreFoundation                      0x00007fff38a45bd3 CFRunLoopRunSpecific + 499
	34  HIToolbox                           0x00007fff3759b65d RunCurrentEventLoopInMode + 292
	35  HIToolbox                           0x00007fff3759b39d ReceiveNextEventCommon + 600
	36  HIToolbox                           0x00007fff3759b127 _BlockUntilNextEventMatchingListInModeWithFilter + 64
	37  AppKit                              0x00007fff35c0bba4 _DPSNextEvent + 990
	38  AppKit                              0x00007fff35c0a380 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1352
	39  AppKit                              0x00007fff35bfc09e -[NSApplication run] + 658
	40  AppKit                              0x00007fff35bce465 NSApplicationMain + 777
	41  MachOExplorer                       0x0000000100047a8d main + 13
	42  libdyld.dylib                       0x00007fff6ff527fd start + 1
```

**Comment by DeVaukz** [comment]
This has been fixed for a while in Mach-O Kit.  I need to sync Mach-O Explorer up with the latest Mach-O Kit commit.

**Comment by DeVaukz** [comment]
Should now be fixed with 14e2966fd8c0c60064172ac5cbb9ebff63fbba13
